vnext/Shared/Modules/ExceptionsManagerModule.cpp

```diff
@@ -156,11 +156,12 @@ std::map<std::string, folly::dynamic> ExceptionsManagerModule::getConstants() {
 }
 
 std::vector<facebook::xplat::module::CxxModule::Method> ExceptionsManagerModule::getMethods() {
+  auto redboxHandler = m_redboxHandler;
   return {Method(
               "reportFatalException",
-              [this](folly::dynamic args) noexcept {
```

Comment:
OOC: was the change for a known lifetime issue, or just to capture less/cleanliness? I've seen the pattern of capturing "this" ptr in module method lambdas in a few places, and thought it was kind of sketch, but assumed everything worked out if it was such a prevalent pattern?